<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntIspratniciVnes</class>
 <widget class="QWidget" name="IntIspratniciVnes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>669</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1051</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="10" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Рок за плакање(денови)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Додај </string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="5">
     <widget class="QTableView" name="tableView"/>
    </item>
    <item row="4" column="1">
     <widget class="QDateTimeEdit" name="dateTimeDatum">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="rok_za_plakanje_denovi">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="artikal">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Притисни ЕНТЕР за да те однесе во листата&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDateTimeEdit" name="dateTimeValuta">
      <property name="date">
       <date>
        <year>2000</year>
        <month>3</month>
        <day>1</day>
       </date>
      </property>
      <property name="displayFormat">
       <string>d/M/yyyy</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="3">
     <widget class="QLineEdit" name="komintent">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Притисни ЕНТЕР за да те однесе во листата&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Забелешка</string>
      </property>
     </widget>
    </item>
    <item row="7" column="3" rowspan="4">
     <widget class="QTextEdit" name="zabeleska"/>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Залиха</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="cena_so_ddv"/>
    </item>
    <item row="6" column="3">
     <widget class="QLineEdit" name="zaliha">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="kolicina"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Цена со ДДВ</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Количина</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0" colspan="5">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Вкупен износ без ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="vk_iznos_bez_ddv">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>ДДВ Износ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="vk_ddv_iznos">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Вкупен Износ со ДДВ</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="vk_iznos_so_ddv">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Коминтент</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="sifra_komintent">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Шифра на коминтент</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="sifra_artikal">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Шифра на артикал</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="rabat"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Валута</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:12pt; font-weight:600; color:#0000ff;&quot;&gt;Интерна Испратница - Внес&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string> Внеси</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../Brisi/sterna/Resources/key_f9.png</normaloff>../../../Brisi/sterna/Resources/key_f9.png</iconset>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QPushButton" name="pushButtonA">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Импорт Испратници</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Датум</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Артикал</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Рабат</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Бриши</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>sifra_komintent</tabstop>
  <tabstop>komintent</tabstop>
  <tabstop>dateTimeDatum</tabstop>
  <tabstop>dateTimeValuta</tabstop>
  <tabstop>sifra_artikal</tabstop>
  <tabstop>artikal</tabstop>
  <tabstop>zaliha</tabstop>
  <tabstop>kolicina</tabstop>
  <tabstop>cena_so_ddv</tabstop>
  <tabstop>rabat</tabstop>
  <tabstop>rok_za_plakanje_denovi</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButtonA</tabstop>
  <tabstop>pushButton_4</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>zabeleska</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>vk_iznos_bez_ddv</tabstop>
  <tabstop>vk_ddv_iznos</tabstop>
  <tabstop>vk_iznos_so_ddv</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
